README.md

```diff
@@ -7,80 +7,20 @@ From a high level, blitz-plugin consists of the following tasks/stages:
 
 1. Import `.ome.xml` map files from `org.openmicroscopy:omero-model` (`omero-model.jar`) resources
 2. Import `-types.properties` files from `org.openmicroscopy:omero-model` (`omero-model.jar`) resources
-3. Using the [`omero-dsl-plugin`](https://github.com/ome/omero-dsl-plugin), generate `xx.combined` files
-4. Process and split `xx.combined` files into chosen languages
 
 ### Usage
 
-Build script snippet for use in all Gradle versions:
-
-```groovy
-buildscript {
-    repositories {
-        mavenLocal() // If plugin is locally published
-    }
-    dependencies {
-         classpath 'org.openmicroscopy:blitzplugin:1.0.0'
-    }
-}
-
-apply plugin: "org.openmicroscopy.blitzplugin"
-```
-
-Build script snippet for new, incubating, plugin mechanism introduced in Gradle 2.1:
+Include the following to the top of your _build.gradle_ file:
 
 ```groovy
 plugins {
-    id "org.openmicroscopy.blitzplugin" version "1.0.0"
+    id "org.openmicroscopy.blitz" version "1.0.0"
```

Comment:
1.0.0 seems unlikely. Needs a tip to fill in version accordingly or something?

Reply:
I'm going to change to `x.y.z` across the board so that we don't need to keep them in sync.